i18n/fr.yaml

```diff
@@ -6,11 +6,11 @@ title:
   other: "COVID Shield"
 intro:
   other: "COVID Shield est une solution de notification d'exposition à la COVID-19 libre d'accès, conçue dans le respect absolu de la confidentialité. Elle a été créée par un groupe de bénévoles dans le but d'aider les Canadiens et le reste du monde à reprendre le travail dans de bonnes conditions de sécurité."
-disclaimer_title:
+download_title:
   other: "Comment puis-je télécharger et utiliser COVID Shield?"
-disclaimer_message:
+download_message:
   other: "COVID Shield n'est pas disponible actuellement pour usage, ni pour téléchargement."
```

Comment:
Change to

`Alerte COVID, la version canadienne officielle de COVID Shield, est maintenant disponible pour <a href="https://apps.apple.com/fr/app/id1520284227>iOS</a> et <a href="https://play.google.com/store/apps/details?id=ca.gc.hcsc.canada.stopcovid&hl=fr>Android</a>.`

Reply:
All done!